### PR TITLE
Various changes (nds-bootstrap 0.39.0 and minor corrections mostly)

### PR DESCRIPTION
**Pull request by spellboundtriangle**
Please read the changes and Apr 21 commits

### CODE REVIEW

File: pages/_en-US/nds-bootstrap/faq.md [hunk 1]
```diff
@@ -40,15 +40,12 @@ You can get nightly builds for nds-bootstrap [here](https://github.com/TWLBot/Bu
...
-#### Is there anything else I should know about cheats?
-You can set heap shrink to `Hi` in settings for slightly better cheat support.
-
 For more info on cheats, look [here](https://wiki.ds-homebrew.com/ds-index/retail-roms#action-replay-cheats).
 
 #### nds-bootstrap troubleshooting
 If you get issues with using the latest version of nds-bootstrap, here's how you could troubleshoot it.
 
-1. Run everything in 67 MHz, DS Mode, with 32 kHz of audio, etc. Basically, keep everything to DS settings. Don't try to improve it with DSi stuff
+1. Run the game with different settings, including changing the ARM9 CPU Speed, DS / DSi Mode, sound quality, Card read DMA, etc
```
Comment (Epicpkmn11) [inline pages/_en-US/nds-bootstrap/faq.md:48]: "Card read DMA" should probably be either "Card Read DMA" or "card read DMA", the former is probably more fitting with the other things here.

Other than that (and the same thing on testing.md#17) this looks good to me.

File: pages/_en-US/twilightmenu/controls.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ description: Controls for using TWiLight Menu++
         - <kbd>Up</kbd> / <kbd>Down</kbd>: Increase / Decrease selected value
         - <kbd>Left</kbd> / <kbd>Right</kbd>: Select a value
         - <kbd class="face">A</kbd> / <kbd class="face">B</kbd>: Return to RAM Viewer/Editor at specified address
+ - The in-game menu is not accessible in DSi Mode, so attempting to use the button combo  in DSi Mode will instead exit the game.
```
Comment (Epicpkmn11) [inline pages/_en-US/twilightmenu/controls.md:62]:
```suggestion
 - The in-game menu is not accessible in DSi Mode, so attempting to use the button combo in DSi Mode will instead exit the game.
```
Oh, double space here

**Comment by spellboundtriangle** [comment]
I think I've added your changes correctly, so is it now ready to be merged?

**Comment by Epicpkmn11** [comment]
Looks good to me and no one else has said anything so I'll merge it

**Pull request merged**